modular_zubbers/code/modules/hydroponics/code/grown/rocks.dm

```diff
@@ -91,3 +92,421 @@
 
 /datum/reagent/consumable/nutriment/mineral
 	taste_description = "rocks and stones"
+
+
+//Rockfruits evolutions? OREFRUITS!//
+//Sand - Base tier breaks into 4 trees ('energy', Precious, Metal, Miscmats)
+/obj/item/seeds/sandfruit
+	name = "sandfruit seed pack"
+	desc = "These seeds grow to produce sandfruits."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/seeds.dmi'
+	icon_state = "seed-sandfruit"
+	species = "ore"
+	plantname = "Sandfruits"
+	product = /obj/item/food/grown/shell/sand
+	mutatelist = list(/obj/item/seeds/uraniberry,
+					/obj/item/seeds/aubergine,
+					/obj/item/seeds/ferrotuber,
+					/obj/item/seeds/adamapple)
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1,
+						/datum/reagent/silicon = 0.1)
+	growthstages = 2
+	growing_icon = 'modular_zubbers/code/modules/hydroponics/icons/growing_fruits.dmi'
+	genes = list(/datum/plant_gene/trait/chem_cooling)
+
+/obj/item/food/grown/shell/sand
+	seed = /obj/item/seeds/sandfruit
+	name = "sandfruit"
+	desc = "A mutated variant of rockfruits; rough, course and now available everywhere. Just peel it for a core."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/harvest.dmi'
+	icon_state = "orefruit"
+	trash_type = /obj/item/grown/sandfruitcore
+	foodtypes = FRUIT
+	distill_reagent = /datum/reagent/silicon
+
+/obj/item/grown/sandfruitcore
+	seed = /obj/item/seeds/sandfruit
+	name = "sandfruit core"
+	desc = "A very fragile sandfruit core, literally composed of dozens of particles of sand... don't store in pockets."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/harvest.dmi'
+	icon_state = "sandfruit"
+	custom_materials = list(/datum/material/glass =SHEET_MATERIAL_AMOUNT*0.2)
```

Comment:
Inconsistent indendation with the = operator, either have no space or a single space on both sides, and keep it consistent with all the other cases, I reccomend no spaces given the * already has none

Reply:
It's consistent within this file, but the rest of the codebase uses either list(/datum/material/glass=SHEET_MATERIAL_AMOUNT) or list(/datum/material/glass = SHEET_MATERIAL_AMOUNT)

Reply:
Which of the two formats would you rather I put in them all?